frontend/src/lib/components/AppConnect.svelte

```diff
@@ -30,6 +30,9 @@
 				'Create an API key',
 				'Copy your key'
 			]
+		},
+		supabase: {
+			instructions: ['Go to the API Settings of your app to find the project URL and key']
```

Comment:
Would be a little bit nicer with an img :)

Reply:
Done!